build.gradle

```diff
@@ -16,6 +16,8 @@ dependencies {
 
     osgiRuntime project('mucommander-sftp')
     osgiRuntime project('mucommander-rar')
+    osgiRuntime project('mucommander-ovirt')
```

Comment:
the ovirt plugin doesn't seem to be ready yet - can you please rebase on top of the master branch?

Reply:
sure. will do.